traffic_stats/traffic_stats.go

```diff
@@ -506,9 +506,9 @@ func getToData(config StartupConfig, init bool, configChan chan RunningConfig) {
 	if err != nil {
 		errHndlr(err, ERROR)
 	} else {
-		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05+00", lastSummaryTimeStr)
+		lastSummaryTime, err := time.Parse("2006-01-02 15:04:05-05", lastSummaryTimeStr)
```

Comment:
shouldn't it be `-07` and not `-05`?   https://gobyexample.com/time-formatting-parsing

Reply:
hmm...I tested with -05 and that worked, let me try with -07